torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -87,6 +88,26 @@ namespace {
 
 constexpr int64_t kSeedInfoId = -127389;
 
+// Traits related to the return type of the lambda function that wraps the
+// actual implementation inside PythonScope.
+template <class T>
+struct LambdaReturnTypeTrait {
+  using Type = T;
```

Comment:
`Type` is unclear. Also for trait classes we typically follow the STL naming convention of lowercase members.

`raw_type`?

Reply:
Right. `Type` actually represents the return type of the wrapper lambda, given `T` is the return type of the implementation function. So, as mentioned above, what do you think of: `WrapperLambdaReturn<ImplRetType>::type`?

Reply:
If we call the class `RemoveStatus`, then it makes sense to call this `type`, as `RemoveStatus<T>::type` is clear.